deployments/gpu-operator/templates/clusterrole.yaml

```diff
@@ -144,3 +144,13 @@ rules:
 {{- if .Values.operator.cleanupCRD }}
   - delete
 {{- end }}
+# TODO: restrict RBAC for DRA driver
+- apiGroups:
+    - ""
+    - apps
```

Comment:
We probably don't need these two. The current permissions for the "apps" and "" (core) api groups which are already specified in this doc should suffice.